src/platforms/stm32/src/lib/platform_defaultatoms.h

```diff
@@ -23,32 +23,8 @@
 
```

Comment:
It looks to me like we can just get rid of those platform_default_atoms modules for stm32, right?  Can the `stm32` atom just be defined somewhere else?

Reply:
I see no reason not to do that in sys_init_platform(), seems like the perfect place for that.